overview/index.html

```diff
@@ -46,30 +46,32 @@
             <p>
                 Welcome <span id="username" />
             </p>
-            <table class="table">
-                <thead>
-                    <tr>
-                    <th>Name</th>
-                    <th>Repo</th>
-                    <th>Endpoint</th>
-                    <th>SHA</th>
-                    <th>Built</th>
-                    <th>Invocation Count</th>
-                    <th>Replicas</th>
-                    </tr>
-                </thead>
-                <tbody id="items">
-                    <tr id="loading">
-                        <td>Loading...</td>
-                        <td></td>
-                        <td></td>
-                        <td></td>
-                        <td></td>
-                        <td></td>
-                        <td></td>
-                    </tr>
-                </tbody>    
-            </table>
+            <div class="table-responsive-sm table-responsive-md">
```

Comment:
Can you explain how this annotation is working? `table-responsive-sm table-responsive-md`

Reply:
It is breakpoint specific. For small and medium, it will make table responsive by adding horizontal scrolling otherwise it will display the table in full length. 

https://getbootstrap.com/docs/4.0/content/tables/#breakpoint-specific